text/0000-glimmer-components.md

```diff
@@ -0,0 +1,1022 @@
+RFC](https://github.com/emberjs/rfcs/blob/0b32059b3704836e52c906a0ead64ac186c844d8/active/0000-component-unification.md).
+The idea was simple: switch to the superior `<angle-bracket>` syntax of web
+components, and solve all the other problems! Seems easy enough, right?
+
+As you can imagine, it was _not_ that easy. There was a flurry of discussion on
+the original RFC, and many ideas were thrown around. This was seen as the _one_
+chance Ember had to "fix" its component API, and the community did not want to
+get it wrong and lock us into yet _another_ set of painful papercuts. After much
+debate and lots of back and forth with the design, it was ultimately decided
+that attempting to redesign components all at once, monolithically, was too
+much. Instead, the individual ideas from that discussion could be broken out and
+implemented in isolation, in a backwards compatible way, both incrementally
+building a new, well thought out component API _and_ laying the groundwork in
+the framework for future redesigns.
+
+Many of the features from these discussions which are core to Glimmer
+components have already landed in Ember, including: Angle-bracket invocation,
+named arguments, element modifiers, and component managers.
+
+Glimmer components are the final piece of that puzzle. They include the last
+of the major features that were discussed during the original Angle Brackets
+RFC, and holistically those features make a much simpler and more ergonomic
+component API. Taken alone, they are an incremental change. Their individual
+features aren't _that_ much more than what we currently have in Ember today. But
+as a whole they represent the culmination of multiple years of design work and
+discussion by the Ember community, and the collective attention to detail and
+care of all of our community members.
+
+## Terminology
+
+* The **Glimmer VM** is the underlying rendering engine which is used by
+  _Ember.js_ and _Glimmer.js_.
+* **Glimmer.js** is a thin wrapper on top of the _Glimmer VM_ which exposes a
+  much simpler API compared to Ember. Historically it has been used to prove out
+  ideas and implementations before bringing them into Ember via RFC, and has
+  been used to write applications which don't require the full feature set of
+  Ember.
+* **Glimmer components** are a newly proposed component API which draw from the
+  experimental APIs provided in Glimmer.js, and Ember.js via
+  [sparkles-component](https://github.com/rwjblue/sparkles-component).
+* **Classic components** refer to the standard component API at the time of this
+  RFC, which have been available in Ember in some form since v1.
+* **Tracked properties** refer to a new method of change tracking which is being
+  proposed in a separate RFC, parallel to this one.
+
+## Motivation
+
+Glimmer components are a simpler base component class that encourage template
+oriented design, thinner class definitions, and functional/composable component
```

Comment:
Beyond this sentence, I don't see any further discussion of composable design. If we are going to mention that, I think we probably need to discuss it a bit since I don't think it will be obvious to most readers. But, I don't know how important this is to the overall motivation and if we even need to mention it here.